demos/enron-demo.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+
+import palimpzest as pz
+from palimpzest.core.lib.schemas import TextFile
+
+
+class EnronReader(pz.DataReader):
+    def __init__(self, dir: str, labels_file: str | None = None, split: str = "test"):
+        super().__init__(TextFile)
+        self.filepaths = [os.path.join(dir, filename) for filename in os.listdir(dir)]
+        self.filepaths = self.filepaths[:50] if split == "train" else self.filepaths[50:150]
+        self.filename_to_labels = {}
+        if labels_file:
+            with open(labels_file) as f:
+                self.filename_to_labels = json.load(f)
+
+    def __len__(self):
+        return len(self.filepaths)
+
+    def __getitem__(self, idx: int):
+        # get input fields
+        filepath = self.filepaths[idx]
+        filename = os.path.basename(filepath)
+        with open(filepath) as f:
+            contents = f.read()
+        
+        # create item with fields
+        item = {"fields": {}, "labels": {}}
+        item["fields"]["filename"] = filename
+        item["fields"]["contents"] = contents
+        item["labels"] = self.filename_to_labels.get(filename, {})
+
+        return item
+
+if __name__ == "__main__":
+    # create validation data source
+    val_datareader = EnronReader(dir="testdata/enron-eval-medium", labels_file="enron-eval-medium-labels.json", split="train")
```

Comment:
Should we add these files to testdata/ folder? If not, we should at least add them to the download script.

Reply:
good call, I'll do that now